src/app/features/login-page/login-page.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { Router } from '@angular/router';
+import { AuthService } from '../../shared/services/auth.service';
+import { CommonModule } from '@angular/common';
+import { MatSnackBar } from '@angular/material/snack-bar'; 
+
+
+@Component({
+  selector: 'srp-login-page',
+  standalone: true,
+  imports: [ReactiveFormsModule, CommonModule],
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss'],
+})
+export class LoginPageComponent {
```

Comment:
Remove page prefix from the all components `login-page` -> `login`, `dashboard-page` -> `dashboard`